website/test-balance/src/kafka/kafka.consumer.ts

```diff
@@ -61,10 +59,10 @@ export class KafkaConsumer {
       return { error: JSONRPC_ERRORS.InternalError.name };
     }
 
-    if (payload.genesis === this.gearService.genesisHash) {
+    if (payload.genesis === this.gearService.getGenesisHash) {
       try {
         if (await this.dbService.possibleToTransfer(payload.address, payload.genesis)) {
-          result = { result: await this.gearService.transferBalance(payload.address) };
+          result = { result: await this.gearService.transferBalance(payload.address, this.gearService.rootAccount) };
```

Comment:
no way
You should transfer balance not from the root account, but just from account.
Maybe it's better to make this.account as default in transfer balance method. And only in case of transfer to this.account use rootaccount. 
Root account is only needed to top up the account balance.
Maybe we should rename them